packages/manager/modules/manager-pci-common/src/api/hook/useFlavors.ts

```diff
@@ -132,6 +132,23 @@ export const useMergedKubeFlavors = (projectId: string, region: string) => {
         const plan = availability.plans?.find(
           (_plan) => _plan.code === flavor.planCodes.hourly,
         );
+
+        // Bugfix: Issue with selecting the monthly price in the US
+        // -----------------------------------------------------------
+        // Context: Normally, we take the first element of the `pricings` array
+        // to determine the monthly price. However, for US offers, this first
+        // value may not be correct. Instead, we need to specifically look
+        // for the entry containing 'renew' in `capacities`.
+        //
+        // Solution:
+        // - If the first entry (`pricings[0].price`) is greater than 0, we use it.
+        // - Otherwise, we search the `pricings` array for an entry where
+        //   `capacities` includes 'renew'.
+        const checkPriceMonthly =
```

Comment:
```suggestion
        const checkPriceMonthly = addonMonthly?.pricings?.[0].price > 0
             ? addonMonthly.pricings[0]
             : addonMonthly?.pricings?.find((p) => p.capacities.includes('renew'));
```

if `addonMonthly?.pricings?.[0].price > 0` is true that mean it is not undefined